system-int-test/src/cmr/system_int_test/data2/umm_json.clj

```diff
@@ -80,6 +80,7 @@
   [meta]
   (util/remove-nil-keys
    (-> meta
+       (dissoc :granule_count)
```

Comment:
There are many existing tests that are making use of this without granule_count.  Since we are testing the granule_count in granule_counts_search_test.clj, we don't need to test them here.